tests/osaka/eip7692_eof_v1/eip6206_jumpf/test_jumpf_execution.py

```diff
@@ -83,6 +105,100 @@ def test_jumpf_to_self(
     )
 
 
+@pytest.mark.parametrize(
+    "container",
+    [
+        Container(
+            name="1_to_2_arg0",
+            sections=[
+                Section.Code(
+                    Op.CALLF[1] + Op.STOP,
+                ),
+                Section.Code(
+                    Op.PUSH0 + Op.RJUMPI[3] + Op.JUMPF[2] + Op.RETF,
+                    code_outputs=0,
+                ),
+                Section.Code(
+                    Op.SSTORE(slot_code_worked, value_code_worked) + Op.RETF,
+                    code_outputs=0,
+                ),
+            ],
+        ),
+        Container(
+            name="1_to_2_arg1",
+            sections=[
+                Section.Code(
+                    Op.PUSH1[1] + Op.CALLF[1] + Op.STOP,
+                ),
+                Section.Code(
+                    Op.RJUMPI[1]
+                    + Op.RETF
+                    + Op.SSTORE(slot_code_worked, value_code_worked)
+                    + Op.JUMPF[2],
+                    code_inputs=1,
+                    code_outputs=0,
+                ),
+                Section.Code(
+                    Op.RETF,
+                    code_outputs=0,
+                ),
+            ],
+        ),
+        Container(
+            name="1_to_0_to_1",
+            sections=[
+                Section.Code(
+                    Op.ISZERO(Op.SLOAD(slot_code_worked)) + Op.CALLF[1] + Op.STOP,
+                ),
+                Section.Code(
+                    Op.RJUMPI[1]
+                    + Op.RETF
+                    + Op.SSTORE(slot_code_worked, value_code_worked)
+                    + Op.JUMPF[0],
+                    code_inputs=1,
+                    code_outputs=0,
+                ),
+            ],
+        ),
+        Container(
+            name="retf_in_nonreturning",
+            sections=[
+                Section.Code(
+                    Op.PUSH0 + Op.JUMPF[1],
+                ),
+                Section.Code(
+                    Op.RJUMPI[1] + Op.RETF + Op.JUMPF[0],
+                ),
+            ],
+            validity_error=EOFException.INVALID_NON_RETURNING_FLAG,
+        ),
+        Container(
+            name="jumpf_to_returning",
+            sections=[
+                Section.Code(
+                    Op.PUSH0 + Op.JUMPF[1],
+                ),
+                Section.Code(
+                    Op.RJUMPI[1] + Op.RETF + Op.JUMPF[2],
```

Comment:
I can't remember offhand - will this calculate `code_inputs=1` for this section? Do we want it to have 1, so that no implementation flips on different error?

Reply:
Me neither, but I'm guessing not.

Reply:
I think this is the intention of the test: we want a section having `RETF` and `JUMPF` in it and it should fail because of the section being "non-returning" and `JUMPF` targets "returning". But I'm not sure such test make a lot of sense since it obviously fails because of the `RETF`. Maybe I'll just swap the `RETF` and `JUMPF` order so that `JUMPF` is inspected earlier.

Reply:
In the end I added one more test.

Reply:
Oh, ok, I confused myself...